### PR TITLE
Latejoin vox now have an increased chance to carry a virus

### PR DESCRIPTION
**Pull request by Kharkov72**
Vox have pox, you know. It's a scientific fact.

:cl:
 * rscadd: Latejoin vox have a higher chance to have pox.

### CODE REVIEW

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -298,7 +298,7 @@
 
 /mob/new_player/proc/DiseaseCarrierCheck(var/mob/living/carbon/human/H)
 	// 10% of players are joining the station with some minor disease
-	if(prob(10))
+	if(prob(10) || isvox(H) && prob(10))
```
Comment (boy2mantwicethefam) [inline code/modules/mob/new_player/new_player.dm]: Add some parantheses, you can never be too sure
if(prob(10) || (isvox(H) && prob(10)))

Comment (Kurfursten) [inline code/modules/mob/new_player/new_player.dm]: This doesn't do what you think it does. What this is going to do is make Vox have the same chance to have a minor disease as before (10%), but lower the chance for humans to arrive carrying a disease to 1%.

Either bad code or trying to sneak a nerf to new viro under a different name.

**Comment by Kurfursten** [comment]
/vg/station: latejoin diseases are bad and far too common
also /vg/station: it's okay when it's grudgecode

**Comment by SkowronX** [comment]
delet this

**Comment by DeityLink** [comment]
Frankly latejoin diseases haven't really been much of a problem so far. Especially now that medbay is getting the hang of nuviro. I'm all for stealthy memes like that.

**Comment by ghost** [comment]
This isn't grudgecode so much as it is a means of rectifying the issue that non-human races aren't supposed to have overwhelming advantages over humans.

**Comment by SonixApache** [comment]
100% of vox should spawn with bird flu

**Comment by kilozombie** [comment]
Attention. All Vox must report to Medbay for their innoculations.

image

File: code/modules/mob/new_player/new_player.dm (hunk 1 above)
Comment (Exxion) [reply code/modules/mob/new_player/new_player.dm]: This is incorrect. It doesn't change the normal chance but it rolls twice for vox, which if I didn't fuck up due to being tired would give them a 19% chance

Comment (Exxion) [reply code/modules/mob/new_player/new_player.dm]: Wrong button
Anyway I made a comment here before but it was wrong so ignore it

**Comment by Gringonius** [comment]
> 
> 
> This isn't grudgecode so much as it is a means of rectifying the issue that non-human races aren't supposed to have overwhelming advantages over humans.

Where's the isplasmaman(H)?

**Comment by Kharkov72** [comment]
plasmamen and all forms of skeletons really don't have organs, so how would they have a virus

**Comment by DeityLink** [comment]
They just do

**Comment by ghost** [comment]
> 
> 
> plasmamen and all forms of skeletons really don't have organs, so how would they have a virus

it infects their bone marrow

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Plasmamen have organs

**Comment by Kharkov72** [comment]
judging by the sheer volume of viruses we have at the moment, this is probably something we don't need right now. I'll revisit it later

**Pull request closed without merging**